openupgrade_scripts/scripts/account/14.0.1.1/pre-migration.py

```diff
@@ -330,6 +330,27 @@ def add_move_id_field_account_payment(env):
     )
 
 
+def add_edi_state_field_account_move(env):
+    """
+    Module account_edi: Created edi_state column and set the default value is false
+    """
+    if not openupgrade.column_exists(env.cr, "account_move", "edi_state"):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            ALTER TABLE account_move
+            ADD COLUMN edi_state varchar
+            """,
+        )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move
+        SET edi_state = false
+        """,
+    )
```

Comment:
@pedrobaeza no need for this code, because `edi_state` is set as 'false' in database
CC @tranngocson1996

Reply:
PR fix : https://github.com/OCA/OpenUpgrade/pull/3119

Reply:
If nothing is done explicitly, it's set to Null, not False.

Reply:
Isn't this a boolean field?

Reply:
@pedrobaeza yes
![Screenshot from 2022-01-27 17-02-58](https://user-images.githubusercontent.com/43790414/151336863-4a056bc7-cbe3-40f6-9122-e4d8dbc80ccc.png)

Reply:
OK, then indeed this is incorrect. I thought it was a boolean one.